pkg/config/default_test.go

```diff
@@ -1,11 +1,39 @@
 package config
 
 import (
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func TestDefaultMaskedPaths(t *testing.T) {
```

Comment:
This should likely be a new file `default_linux_test.go` as this test can only be run under linux.